engine/src/main/resources/application.properties

```diff
@@ -120,13 +120,14 @@ quarkus.log.category."com.redhat.cloud.notifications.health.KafkaConsumedTotalCh
 reinject.enabled=false
 
 %test.ob.enabled=true
-ob.bridge.uuid=<uuid-of-the-bridge instance>
+ob.bridge.name=<name-of-the-bridge instance>
 ob.token.client.id=<id of the service account>
 ob.token.client.secret=<secret of the service account>
 
 quarkus.rest-client.ob.url=https://bridge.acme.org
 quarkus.rest-client.kc.url=https://keycloak.acme.org
 quarkus.cache.caffeine.kc-cache.expire-after-write=PT120s
+quarkus.cache.caffeine.from-open-bridge-history-filler.expire-after-write=PT24H
```

Comment:
@pilhuhn How long do you think we should cache the seen ids?

Reply:
I have no idea and we should perhaps watch the returned a list for a while for the ratio of new entries vs existing ones. If most are new, we can lower that to 1h.
Perhaps @danielezonca knows if they expire entries after a fixed amount of time and what the queue length setting on their side is.

Reply:
Keep in mind though, that if there is a sporadic error, it will stay for a long time until the queue is full and the error would be overwritten. If we expire it on our side, we would need to do a little more work, but it would not harm too much.

Reply:
Currently there is no expiration so every time you invoke the error endpoint, you get back the last N errors (default should be `100`) without any expiration time (so they can be quite old)

Reply:
Thanks @danielezonca!

It might be better to rely on a DB table to store the seen ids on our side then.